device_doctor/test/src/ios_device_test.dart

```diff
@@ -52,11 +52,12 @@ void main() {
       Map<String, List<HealthCheckResult>> results = await deviceDiscovery.checkDevices(processManager: processManager);
       expect(results.keys.length, equals(1));
       expect(results.keys.toList()[0], 'ios-device-abcdefg');
-      expect(results['ios-device-abcdefg'].length, equals(4));
+      expect(results['ios-device-abcdefg'].length, equals(5));
```

Comment:
Can we make this more specific? like validating a certain check returns true and some other false? as it is this type of test requires changes every time a new test check is added/removed and the indexes do not tell very much about what check is being executed.

Reply:
Expanded to include each test info.